app/src/main/res/values/strings.xml

```diff
@@ -42,10 +42,10 @@
     <string name="microscope_quality_snackbar_error">An error occurred while saving your microscope quality data. Please try again.</string>
     <string name="microscope_quality_snackbar_retry">TRY AGAIN</string>
     <!-- CaptureImage -->
-    <string name="capture_activity_title">Capture photo</string>
+    <string name="capture_activity_title">Capture a sample</string>
```

Comment:
I am not sure this is correct though. A sample contains multiple captured photos/images, according to our domain entities: https://github.com/theaiscope/GDD-app/blob/20117b43e48c68bc6c9c21420fe450161c4419e6/app/src/main/java/net/aiscope/gdd_app/model/SampleModels.kt

Reply:
'image' now